packages/wrangler/turbo.json

```diff
@@ -65,7 +65,8 @@
 				"WRANGLER_IMPORT",
 				"MINIFLARE_IMPORT",
 				"CLOUDFLARE_ACCOUNT_ID",
-				"CLOUDFLARE_API_TOKEN"
+				"CLOUDFLARE_API_TOKEN",
+				"WRANGLER_E2E_TEST_FILE"
```

Comment:
Does this not remove the benefit, and make turbo consider that env var as part of the test context?

Reply:
This is what we want. The tests are affected by this env var - e.g. if you change which test file to run it shouldn't just hit the cache of the last test file. Without this we would never even pass this env var to the task.

Where we want the cache hits is in the `build` tasks, which don't have this env var listed as an input, so ignore it when working out whether it is a cache hit or not.